src/DynamoCoreWpf/UI/InOutPortPanel.cs

```diff
@@ -48,8 +54,13 @@ protected override Size MeasureOverride(Size constraint)
                     // Default behavior of getting each child to measure.
                     child.Measure(constraint);
                 }
-                catch(System.Exception)
+                catch(XamlParseException e)
```

Comment:
so we will not catch any other Exception types correct ?

Reply:
correct, those other exceptions are not expected and will crash.